pkgs/applications/radio/qradiolink/default.nix

```diff
@@ -1,51 +1,64 @@
-{ stdenv, fetchFromGitHub, alsaLib, boost
-, qt4, libpulseaudio, codec2, libconfig
-, gnuradio, gr-osmosdr, gsm
-, libopus, libjpeg, protobuf, qwt, speex
-} :
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, libpulseaudio
+, libconfig
+, gnuradio
+, gnuradioPackages
+, gsm
+, libopus
+, libjpeg
+, libsndfile
+, libftdi
+, qwt
+, protobuf
+, speex
+, speexdsp
+}:
 
-let
-  version = "0.5.0";
-
-in stdenv.mkDerivation {
+mkDerivation rec {
   pname = "qradiolink";
-  inherit version;
+  version = "0.8.5-rc4";
```

Comment:
Does the new version even build with 0.8.5? Otherwise we could just wait with the update until we have a working gr-3.8.

Reply:
> Does the new version even build with 0.8.5?

The new version of qradiolink? It builds and it's functional if built with gnuradio 3.8. I just wanted to do most of the update myself in this PR because it needed a lot of refactoring and it illustrates the usage of `gnuradio.callPackage`. I won't mind waiting for a non-rc release.

Reply:
Let's wait with the upgrade of qradiolink at least until we have a working gr-3.8 (maybe with the next PR then?). Otherwise it breaks qradiolink.

Reply:
I don't understand why you say it breaks something. qradiolink 0.8.5-rc5 is built with gnuradio 3.8 and it works.

Reply:
Gnuradio 3.8 also launches and seems functional..